lib/worker.js

```diff
@@ -1,6 +1,5 @@
 const os = require('os')
 const path = require('path')
-const exec = require('child_process').exec
```

Comment:
removing the local process check for "stuck" local workers is a huge win.  This was really brittle.